cluster/examples/kubernetes/ceph/csi/template/cephfs/csi-cephfsplugin.yaml

```
@@ -105,6 +105,9 @@ spec:
               mountPath: /tmp/csi/keys
             - name: host-run-mount
               mountPath: /run/mount
+            - name: etc-selinux
```

Can this be optional? In non-openshift or other non-selinux distros it wouldn't be needed

---

@travisn that make sense,  anything specifically in your mind to make this optional mount?

---

Can you use a conditional like this? Then in the operator code that starts the CSI driver it would need to detect if in an environment where needed (or read it from a CSI setting).
```suggestion
            {{ if .SELinuxEnabled }}
            - name: etc-selinux
              mountPath: /etc/selinux
              readOnly: true
            {{ end }}
```

---

yea, so in the latesr verison of this patch a condition is introduced which read from the csi setting.